From 7577b8cf676f7d38363052e2bc8da931af1cc279 Mon Sep 17 00:00:00 2001
From: Jeremy Koritzinsky <jekoritz@microsoft.com>
Date: Thu, 30 Sep 2021 16:04:21 -0700
Subject: [PATCH] Produce symbols archives.

Contributes to #49026

https://github.com/dotnet/runtime/pull/59843 handles adding the patch to the runtime repo.
---
 .../Microsoft.NETCore.App.Runtime.Composite.sfxproj             | 2 +-
 .../Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.sfxproj | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.Composite.sfxproj b/src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.Composite.sfxproj
index a3f364815d6..5fe3ba42252 100644
--- a/src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.Composite.sfxproj
+++ b/src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.Composite.sfxproj
@@ -7,7 +7,7 @@
     <ArchiveName>dotnet-runtime-internal-composite</ArchiveName>
     <OverridePackageId>$(SharedFrameworkName).Composite</OverridePackageId>
     <OverridePackageId Condition="'$(PgoInstrument)' != ''">$(SharedFrameworkName).Composite.PGO</OverridePackageId>
-    <GenerateSymbolsArchive>true</GenerateSymbolsArchive>
+    <CreateSymbolsArchive>true</CreateSymbolsArchive>
     <SymbolsArchiveName>dotnet-runtime-composite-symbols</SymbolsArchiveName>
     <UseTemplatedPlatformManifest>true</UseTemplatedPlatformManifest>
     <ForcePublishReadyToRunComposite>true</ForcePublishReadyToRunComposite>
diff --git a/src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.sfxproj b/src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.sfxproj
index 4fa48f4e0c9..1d43304d72b 100644
--- a/src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.sfxproj
+++ b/src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Runtime.sfxproj
@@ -8,7 +8,7 @@
     <InstallerName Condition="'$(TargetOS)' != 'OSX'">dotnet-runtime</InstallerName>
     <InstallerName Condition="'$(TargetOS)' == 'OSX'">dotnet-runtime-internal</InstallerName>
     <OverridePackageId Condition="'$(PgoInstrument)' != ''">$(SharedFrameworkName).PGO</OverridePackageId>
-    <GenerateSymbolsArchive>true</GenerateSymbolsArchive>
+    <CreateSymbolsArchive>true</CreateSymbolsArchive>
     <SymbolsArchiveName>dotnet-runtime-symbols</SymbolsArchiveName>
     <VSInsertionShortComponentName>NetCore.SharedFramework</VSInsertionShortComponentName>
     <UseTemplatedPlatformManifest>true</UseTemplatedPlatformManifest>
-- 
2.29.2

